git/git.go

```diff
@@ -103,31 +103,6 @@ const (
 
 type remoteSorter []Remote
 
-// NewConfig creates a new configuration. The username and email are the only
-// required config fields.
-func NewConfig(username, email string) Config {
-	return Config{
-		Username: username,
-		Email:    email,
-	}
-}
-
-// NewConfigWithPath calls NewConfig but also sets the git program path.
-func NewConfigWithPath(username, email, programPath string) Config {
-	log.Debug().
-		Str("action", "NewConfigWithPath()").
-		Str("path", programPath).
-		Msg("Make new config.")
-	config := NewConfig(username, email)
-	config.ProgramPath = programPath
-	return config
-}
-
-// NewWrapper creates a new wrapper.
-func NewWrapper(user, email string) (*Git, error) {
```

Comment:
Only new wrapper was used on tests, these functions confused me on how was I supposed to create a git wrapper.